src/main/java/pokecube/core/client/MoveSound.java

```diff
@@ -42,7 +42,7 @@ public void tick()
     {
         this.pos2.set(this.player);
         this.volume = MoveSound.getVolume(this.pos1, this.pos2, this.volumeScale);
-        if (this.time++ > 100 || this.volume == 0) this.donePlaying = true;
+        if (this.time++ > 100 || this.volume == 0) this.repeat = true;
```

Comment:
This file needed to be entirely looked into, so I am not sure if this fix would work.